recipes/libmysqlclient/all/conanfile.py

```diff
@@ -96,15 +156,20 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
         os.mkdir(os.path.join(self.package_folder, "licenses"))
```

Comment:
We usually do not need this 🤔

Reply:
It's a workaround because `cmake.install` puts LICENSE file in package_folder. Another alternative, but same result is call self.copy(LICENSE...) and remove the installed LICENSE in package_folder.